libs/core/src/lib/popover/popover-mobile/popover-mobile.component.ts

```diff
@@ -0,0 +1,116 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ElementRef,
+    Inject,
+    OnDestroy,
+    OnInit,
+    Optional,
+    TemplateRef,
+    ViewChild,
+    ViewEncapsulation,
+} from '@angular/core';
+import { Subscription } from 'rxjs';
+import {
+    MobileModeBase,
+    MOBILE_MODE_CONFIG,
+    MobileModeControl,
+    MobileModeConfigToken
+} from '@fundamental-ngx/core/mobile-mode';
+import { PopoverInterface, POPOVER_COMPONENT } from '../popover.interface';
+import { takeUntil } from 'rxjs/operators';
+import { DialogService } from '@fundamental-ngx/core/dialog';
+
+@Component({
+    selector: 'fd-popover-mobile',
+    templateUrl: './popover-mobile.component.html',
+    styleUrls: ['./popover-mobile.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None,
+})
+export class PopoverMobileComponent extends MobileModeBase<PopoverInterface> implements OnInit, OnDestroy {
+    /** @hidden
+     * from mobile class can not prefix _,
+     * to avoid build issues
+    */
+    childContent: TemplateRef<any> = undefined;
+
+    /** @hidden */
+    @ViewChild('dialogTemplate')
+    _dialogTemplate: TemplateRef<any>;
+
+    /** Current popover title */
+    title: string;
+
+    /** Dialog body content */
+    viewBody: TemplateRef<any>;
+    /** Dialog footer content */
+    viewFooter: TemplateRef<any>;
+
+    /** @hidden */
+    private _subscriptions = new Subscription();
+
+    constructor(
```

Comment:
`@hidden` annotation?

Reply:
It's a typedoc comment

Reply:
`/**@hidden */` you need a space in front of @hidden, won't work without it